test/toolkits/style_transfer/test_style_transfer_data_iterator.cxx

```diff
@@ -0,0 +1,42 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#define BOOST_TEST_MODULE
+
+#include <boost/test/unit_test.hpp>
+#include <core/util/test_macros.hpp>
+
+#include <toolkits/style_transfer/style_transfer_data_iterator.hpp>
+
+#include <string>
+
+#include "utils.hpp"
```

Comment:
Let's continue to avoid header includes with quotation marks

Reply:
Turned out to be a bit tricky for tests. Don't know how to address that but I added an include directories to the current CMakeLists. Maybe that's not the answer. Let me know what you think!